vertx-web/src/main/java/io/vertx/ext/web/impl/ForwardedParser.java

```diff
@@ -0,0 +1,177 @@
+//Heavily influenced from spring forward header parser
```

Comment:
I believe for copyright compliance with Eclipse foundation we need a header here like in the other source files.

Reply:
Done